test/e2e/lib/components/shopping-cart-widget-component.js

```diff
@@ -43,31 +43,4 @@ export default class ShoppingCartWidgetComponent extends AsyncBaseContainer {
 			}
 		}
 	}
-
-	async removeDomainRegistraion( domain ) {
-		return this.remove( 'domain_reg', domain );
-	}
-
-	async removeDomainMapping( domain ) {
-		return this.remove( 'domain_map', domain );
-	}
-
-	async remove( type, name ) {
-		const cartBadgeLocator = by.css( '.cart__count-badge' );
-
-		const present = await driverHelper.isElementLocated( this.driver, cartBadgeLocator );
```

Comment:
My guess is that it wasn't working properly because of this line. The cart is updating its content async and needs a couple of seconds to append the item count badge. `isElementLocated` checks the element presence immediately, so I think using `waitUntilElementLocatedAndVisible( this.driver, cartBadgeLocator )` instead should work in this case.

Reply:
Implemented in https://github.com/Automattic/wp-calypso/pull/53114